math/fastor/include/algebra/math/impl/fastor_transform3.hpp

```diff
@@ -0,0 +1,234 @@
+/** Algebra plugins, part of the ACTS project
+ *
+ * (c) 2022 CERN for the benefit of the ACTS project
+ *
+ * Mozilla Public License Version 2.0
+ */
+
+#pragma once
+
+// Project include(s).
+#include "algebra/math/common.hpp"
+#include "algebra/qualifiers.hpp"
+
+// Fastor include(s).
+#include <Fastor/Fastor.h>
+
+// System include(s).
+#include <cstddef>
+
+namespace algebra::fastor::math {
+
+/** Transform wrapper class to ensure standard API within differnt plugins
+ *
+ **/
+template <typename scalar_t, typename matrix_actor_t>
+struct transform3 {
+  /// @name Type definitions for the struct
+  /// @{
+
+  /// Array type used by the transform
+  template <std::size_t N>
+  using array_type = Fastor::Tensor<scalar_t, N>;
+  /// Scalar type used by the transform
+  using scalar_type = scalar_t;
+
+  /// 3-element "vector" type
+  using vector3 = array_type<3>;
+  /// Point in 3D space
+  using point3 = vector3;
+  /// Point in 2D space
+  using point2 = array_type<2>;
+
+  /// 4x4 matrix type
+  using matrix44 = Fastor::Tensor<scalar_type, 4, 4>;
```

Comment:
You could probably also write these as ```4UL```, but that's really just a minor detail

Reply:
I'll make the change.

Reply:
Yes, it's really just a minor readability thing that we are also not doing consistently anyway. Since the compiler will figure this out, and given that we are a bit short on time, let's just skip this comment

Reply:
Sure thing.